docs/src/developer-guide/working-with-rules.md

```diff
@@ -377,6 +377,13 @@ context.report({
 });
 ```
 
+#### Conflicting Fixes
+
+Conflicting fixes are fixes that apply different changes to the same part of a string.
+There is no way to specify which of the conflicting fixes is applied.
+
+For example, if two fixes want to modify characters 0 through 5, only one is applied.
```

Comment:
Maybe this sentence should be before the previous sentence, so that we first explain that only one fix can be applied and then that there is no way to specify which one.

Reply:
i think it makes sense to keep in it's current location b/c this way all of the explanation occurs before the example, rather than breaking up the explanation aroudn the example.